eth2-staker-checklist.md

```diff
@@ -0,0 +1,67 @@
+# Launchpad ENV variables
+
+- `ETH1_NETWORK_NAME`
+- `ETH2_NETWORK_NAME`
+
+----
+
+# Eth2 Staker Checklist
+
+## Hardware
+
+### Recommended
+
+- We recommend you run the Validator Client (VC) on a separate machine from your Beacon Node (BN). This is to ensure your keystore(s) and password(s) are as secure as possible. Read [here](https://blog.ethereum.org/2019/11/27/validated-staking-on-eth2-0/) to understand more about the architecture.
+- In order to process incoming validator deposits from the eth1 chain, you'll need to run an eth1 client in parallel to your eth2 client. While it is possible to use a third-party service like Infura, we recommend running your own client in order to ensure the network stays as decentralised as possible.
+    - For example, you need at least 140 GB SSD to run geth fast sync on *mainnet*.
+
+## Keystore
+1. - [ ] I have securely generated and saved my keystore(s).
+2. - [ ] I have backed up my mnemonic.
+3. - [ ] I have set a strong password for my keystore(s)
+4. - [ ] I have backed up my password. 
+
+## Security
```

Comment:
@sachayves suggested recommending VPN, but I have some concerns about that the latency may deteriorate the performance. Any thoughts?

Reply:
A VPN would go in a "Privacy" section because using a VPN means trusting your VPN provider instead of trusting your ISP. There are shady VPNs that might sell your data.

But being DOS-ed out of validation might be an issue without keeping your IP private.